requirements.txt

```diff
@@ -28,7 +28,7 @@ git+https://github.com/mitodl/django-guardian@de7ca5c39dfd0870067497f5843581e9cd
 git+https://github.com/mingchen/django-cas-ng@69463fe7c23f025be7165c2967f4ceb983d60472#egg=django-cas-ng==3.4.3-odl
 
 # Importer application requirements
-xbundle==0.3.0
+git+https://github.com/mitodl/xbundle@0e6d87b6d87bf0c366b3e87865df2327d57c6525#egg=xbundle==0.3.1
```

Comment:
My understanding is that that's just an alias for the specified commit hash, and since the repo is an mitodl repo, that additional modifier might be unneccessary?